yolox/core/trainer.py

```diff
@@ -137,7 +137,7 @@ def before_train(self):
         self.optimizer = self.exp.get_optimizer(self.args.batch_size)
 
         if self.amp_training:
-            model, optimizer = amp.initialize(model, self.optimizer, opt_level="O1")
+            model, self.optimizer = amp.initialize(model, self.optimizer, opt_level="O1")
```

Comment:
```suggestion
            model, _ = amp.initialize(model, self.optimizer, opt_level="O1")
```
is better.

this is an inplace modify in apex, so don't worry about it.

Reply:
Got it, thanks!